src/test/ui/macros/issue-105225.stderr

```diff
@@ -0,0 +1,33 @@
+error: argument never used
+  --> $DIR/issue-105225.rs:3:21
+   |
+LL |     println!("{x}", x);
+   |              -----  ^ argument never used
+   |              |
+   |              formatting specifier missing
+   |
+   = note: the formatting string captures that binding directly, it doesn't need to be included in the argument list
+help: Consider removing this
+  --> $DIR/issue-105225.rs:3:21
+   |
+LL |     println!("{x}", x);
+   |                     ^
```

Comment:
```suggestion
error: argument never used
  --> $DIR/issue-105225.rs:3:21
   |
LL |     println!("{x}", x);
   |              -----  ^ argument never used
   |              |
   |              formatting specifier missing
   |
note: the formatting string captures that binding directly, it doesn't need to be included in the argument list
     --> $DIR/issue-105225.rs:3:21
   |
LL |     println!("{x}", x);
   |                ^    - this can be removed
   |                |
   |                this formatting argument captures `x` directly
```

In order to add labels in a note you'd have to use a terrible API (details fudged):

```rust
let mut span: MultiSpan = vec_primary_spans.into(); // this should be the formatting arg
for arg in args {
    span.push_span_label(arg.span, format!("this formatting argument captures `{}` directly", arg.name));
}
for span in duplicate_explicit_arg {
    span.push_span_label(span, "this can be removed".to_string());
}
```